test/integration/imagechange_buildtrigger_test.go

```diff
@@ -82,9 +113,7 @@ func TestSimpleImageChangeBuildTrigger(t *testing.T) {
 	}
 
 	event = <-watch2.ResultChan()
-	event = <-watch2.ResultChan()
-
-	updatedConfig, err := openshift.Client.BuildConfigs(testNamespace).Get(config.Name)
+	updatedConfig, err = openshift.Client.BuildConfigs(testNamespace).Get(config.Name)
 	if err != nil {
 		t.Fatalf("Couldn't get BuildConfig: %v", err)
 	}
```

Comment:
I'd copy here, since you already have the `updatedConfig` object check for the `LastTriggeredImageID`:
```
if updatedConfig.Triggers[0].ImageChange.LastTriggeredImageID != "ref-2" {
 t.Errorf("Expected imageID ref-1, got %s", updatedConfig.Triggers[0].ImageChange.LastTriggeredImageID)
 }
```